batch/batch/driver/instance_collection/pool.py

```diff
@@ -34,6 +35,46 @@
     'Number of scheduling loop executions per pool',
     ['pool_name'],
 )
+SCHEDULING_LOOP_RUNS_UNUSED_FREE_CORES = pc.Counter(
+    'scheduling_loop_runs_unused_free_cores',
+    'Number of scheduling loop executions per pool that did not use all of the available free cores',
+    ['pool_name'],
+)
+
+CONTROL_LOOP_RUNS = pc.Counter(
+    'control_loop_runs',
+    'Number of control loop executions per pool',
+    ['pool_name'],
+)
+
```

Comment:
There's an idiom with prometheus metrics to standardize a set of common prefixes (like I use `http` and `sql`). I might have messed this up on some other queries, but can we use `scheduler_` and `autoscaler_` prefixes for these metrics?